qa/selenium-ide/web-server/testCases/Vehicles

```diff
@@ -84,3 +84,4 @@
 </tbody></table>
 </body>
 </html>
+<!--covers test TST-18-->
```

Comment:
I think it's auto generated. The way I have it now basically adds a comment in the script that appears in the Selenium IDE UI. I'll see how it handles meta tags though, and if it doesn't break, go with that.